daemon/daemon.go

```diff
@@ -346,6 +345,22 @@ func (d *daemon) grpcOpts() ([]grpc.ServerOption, error) {
 	return []grpc.ServerOption{}, nil
 }
 
+func (d *daemon) Sleep() error {
+	ticker := time.NewTicker(5 * time.Hour) // every 5 hour check teams on each event
```

Comment:
Either pass this value on the config file or document it somewhere if we agree to leave it here as static value. We are going  to forget about it if left.

Reply:
yeah, exactly.

Reply:
I think, documenting it would be enough instead of passing it through config file.